tails_files/install.sh

```diff
@@ -21,6 +21,13 @@ DESKTOP=$HOMEDIR/Desktop
 ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
 SSH_ALIASES=false
 
+# check for Tails 2.x
+source /etc/os-release
```

Comment:
Why `source` and not `grep`? Altering the environment that `install.sh` runs in should be done carefully. Since this is a read-only operation, `grep` seems the safer option.